scene/gui/text_edit.cpp

```diff
@@ -7000,6 +7000,11 @@ int TextEdit::_get_column_pos_of_word(const String &p_key, const String &p_searc
 				col = p_search.findn(p_key, p_from_column);
 			}
 
+			// if can't find any, just early break to improve performance
```

Comment:
```suggestion
			// If not found, just break early to improve performance.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide), and grammar